File: 0976. Largest Perimeter Triangle/Solution.py
class Solution:
    def largestPerimeter(self, nums):
        """
        type nums: List[int]
        rtype: int
        """
        nums.sort()
        for x in range(len(nums) - 1, 1, -1):
            print(x)
            if nums[x - 1] + nums[x - 2] > nums[x]:
                return nums[x - 1] + nums[x - 2] + nums[x]
        return 0

File: 0976. Largest Perimeter Triangle/Solution.cpp
#include <algorithm>
#include <vector>

using std::vector;

class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int res = 0;
        for (int i = nums.size() - 1; i >= 2; i--)
        {
            if (nums[i - 2] + nums[i - 1] > nums[i])
                return nums[i] + nums[i - 1] + nums[i - 2];
        }
        return res;
    }
};